add: Central Command rework

ItsMeJaba opened the pull request:
## Ссылка на предложение/Причина создания ПР
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
https://discord.com/channels/617003227182792704/755125334097133628/1209229547837464587
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Описание
Я вот тут наделал всякого
![2024-02-12 22 11 42](https://github.com/ss220-space/Paradise/assets/91949115/f15b6a6e-7522-43ec-a2a9-0faf9038b7a3)

# ОБР
* Оружейная ОБР вытянута в единый коридор
![2024-01-14 23 23 56](https://github.com/ss220-space/Paradise/assets/91949115/653372d4-b6d9-4860-b3e8-c4b8ea59bc1f)

* Брифинговая ОБР стала более открытой и удобной, как для передвижения, так и в плане организации мест
![image](https://github.com/ss220-space/Paradise/assets/91949115/c1189e0c-6e56-4baf-9771-a3bba447852e)

* Создан "Плац" с карантинной карантинной блокировкой (выступающие стены являются рескином шаттеров, которые скрывают турели)
![2024-01-14 23 24 47](https://github.com/ss220-space/Paradise/assets/91949115/ef4280b0-a2b0-44b2-be6e-1daa03b5a9c6)

* Медицинский блок был расширен, возвращена машина клонирования.
![2024-01-15 02 30 36](https://github.com/ss220-space/Paradise/assets/91949115/5ed999a8-8a88-40ad-96aa-e3f671d0dd18)

* Добавлена "жилая" зона ОБР
![image](https://github.com/ss220-space/Paradise/assets/91949115/6f1fe250-aa81-4c38-a74a-87fac1c44129)

* Добавлен одноразовый дроппод НТ.
Не содержит турелей и других опасных элементов, просто падающая с потолка бандура
![2024-01-15 01 00 38](https://github.com/ss220-space/Paradise/assets/91949115/eeab9871-8200-4a80-a358-5c8575edd161)

# Отдел снабжения ЦК
![2024-01-16 02 26 27](https://github.com/ss220-space/Paradise/assets/91949115/7519a7d3-db75-48fb-8068-1a5df1015bb8)

* Гамма оружейная смещена в отдел снабжения
* Подставка под медали была изменена на вендомат со спрайтом кейса для медалей капитана
* Добавлены вендоматы со снаряжением медиков, инженерным и уборщика. (Имеют в себе все снаряжение, что не лежит на подставках и коробки с соответствующими хардсьютами)
* Доабвлен вендомат с коробками, содержащие полный набор ЕРТ Эмбер и Ред, а также карточки с самозаписью
* Добавлен вендомат с ресурсами
* Все снаряжение и мехи, что занимали северную "стоянку" помещены в вендоматы
* Добавлен вендомат с пресетами ящиков и сумок.
Имеет 3 вида ящиков: доставка НТ, СБ, оружейный
Наборы: подавления бунтов, на случай военных действий, пауков террора, блоба, лозы и строительного ивента

# Архив
![2024-01-14 23 27 59](https://github.com/ss220-space/Paradise/assets/91949115/b00d557c-46aa-4cba-b37b-12d23912cdbd)

* Имеет своего архивариуса - мимика с фракцией НТ (НЕ АГРЕССИВЕН)
* Помимо рандомных постеров и книг содержит несколько бумаг со старыми мемами, объединенные тематикой факсов и офисной рутины ЦК

# Офисы
![2024-01-14 23 29 11](https://github.com/ss220-space/Paradise/assets/91949115/c913f812-1369-489a-9d31-38cb4fc25318)

* Старая наработка @alexskh ID:697002434240708608, немного измененная

# Офис маппера
![2024-01-14 23 34 26](https://github.com/ss220-space/Paradise/assets/91949115/5699a57a-7c8e-46b4-b167-7e7d5cde82ba)

* Помещение юмористического характера со специально допущенными ошибками

# Шаттлы
16 версий ferry шаттлов
![2023-11-18 20 50 06](https://github.com/ss220-space/Paradise/assets/91949115/ce101598-ea46-4794-9d02-21543e38aa93)

3 версии admin шаттлов
![Безымянный](https://github.com/ss220-space/Paradise/assets/91949115/c280715a-783b-4bfe-a7ed-1360a60b3341)
![2023-11-18 20 52 08](https://github.com/ss220-space/Paradise/assets/91949115/8dba44d4-8086-4a49-b32b-adcdcf0fc966)

# И остальные изменения, что не сильно касаются декора:
* Дуло БСА теперь обладает телепортом прямиком в туалет, чтобы синты не могли использовать их
* В дормах установлены bwonk'ающие мины, что предупреждают любителей кринжа
* Перенос арены бубльгума левее, т.к. при спавне импортированные корабли ломают её

ItsMeJaba committed:
FerryPack

ItsMeJaba committed:
в догонку

ItsMeJaba committed:
Промежуток №2

ItsMeJaba committed:
Проба дроппода

ItsMeJaba committed:
Ребейз карты ручками

ItsMeJaba committed:
Промежуток 3

ItsMeJaba committed:
Промежуток 4

И подчистка

ItsMeJaba committed:
3

ItsMeJaba committed:
Update CentComm.dmm

ItsMeJaba committed:
4

ItsMeJaba committed:
Фиксы

Etrnlmelancholy commented:
Нужны турели как на старом цк об которые экипаж может убиваться

ReloadKey commented:
На обновления ЦК уже очередь, дождитесь всех мержей чтоли или поговорите с димачом.
Также не могу найти эту отсылку, зачем удалили?
![image](https://github.com/ss220-space/Paradise/assets/126337700/fd1ee691-2c47-43d6-b34f-74698da6fea4)

definitelynotspaghetti commented:
Поебень о которой два-три человека могут знать это не отсылка, а хуета

> На обновления ЦК уже очередь, дождитесь всех мержей чтоли или поговорите с димачом. Также не могу найти эту отсылку, зачем удалили? ![image](https://user-images.githubusercontent.com/126337700/284545502-fd1ee691-2c47-43d6-b34f-74698da6fea4.png)

ReloadKey commented:
> Поебень о которой два-три человека могут знать это не отсылка, а хуета
> 
> > На обновления ЦК уже очередь, дождитесь всех мержей чтоли или поговорите с димачом. Также не могу найти эту отсылку, зачем удалили? ![image](https://user-images.githubusercontent.com/126337700/284545502-fd1ee691-2c47-43d6-b34f-74698da6fea4.png)

Имейте уважение к авторам реворков, челяди...

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Мерж ручками

Перенес ручками все, что было добавлено во время простоя + заметил, что правую часть корабля скосило

ItsMeJaba committed:
АААА

АААА

ItsMeJaba committed:
AAAA

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Merge branch 'СС-arm2' of https://github.com/ItsMeJaba/Paradise into СС-arm2

ItsMeJaba committed:
ААА

Я идиот.

ItsMeJaba committed:
Э, а тут что-то происходит?

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Мерж конфликт кривой жабьей лапкой

ItsMeJaba committed:
Ой

Я что-то сделал неделю назад и уже не помню что

ItsMeJaba committed:
Mapper's ref and something what i cant remember

ItsMeJaba committed:
Final?

Я уже не знаю чего мне добавлять, поправил что мог, навел марафета, раскидал мин под кровати

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Conflict.net

Решаем ручками конфликты

ItsMeJaba committed:
Bugfix.da

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Bugfix.da

Ahaha, 3pm fixes

Bugfix.da

Bugfix.da

Bugfix.da

Den109G commented on code/game/machinery/doors/shutters.dm:
```diff
@@ -20,3 +21,9 @@
 	hackable = FALSE
+
+/obj/machinery/door/poddoor/shutters/invincible/fake_r_wall
+	name = "reinforsed wall"
```
```suggestion
	name = "reinforced wall"
```
**GRAMMAR POLICE**
*ouy'er
Если конечно же оно не должно выглядеть как фейковая-фейковая стена.

Den109G commented on _maps/map_files/generic/CentComm.dmm:
```diff
@@ -24803,7 +24836,7 @@
 /obj/machinery/vending/nta{
 	dir = 4;
-	products = list(/obj/item/grenade/flashbang=4,/obj/item/flash=5,/obj/item/flashlight/seclite=4,/obj/item/restraints/legcuffs/bola/energy=8,/obj/item/ammo_box/shotgun=4,/obj/item/ammo_box/shotgun/buck=4,/obj/item/ammo_box/shotgun/rubbershot=4,/obj/item/ammo_casing/shotgun/stunslug=35,/obj/item/ammo_casing/shotgun/ion=14,/obj/item/ammo_casing/shotgun/laserslug=35,/obj/item/ammo_box/speedloader/sho ...
+	products = list(/obj/item/grenade/flashbangs=4,/obj/item/flash=5,/obj/item/flashlight/seclite=4,/obj/item/restraints/legcuffs/bola/energy=8,/obj/item/ammo_box/shotgun=4,/obj/item/ammo_box/shotgun/buck=4,/obj/item/ammo_box/shotgun/rubbershot=4,/obj/item/ammo_casing/shotgun/stunslug=35,/obj/item/ammo_casing/shotgun/ion=14,/obj/item/ammo_casing/shotgun/laserslug=35,/obj/item/ammo_box/speedloader/sh ...
```
```suggestion
	products = list(/obj/item/storage/box/flashbangs=4,/obj/item/flash=5,/obj/item/flashlight/seclite=4,/obj/item/restraints/legcuffs/bola/energy=8,/obj/item/ammo_box/shotgun=4,/obj/item/ammo_box/shotgun/buck=4,/obj/item/ammo_box/shotgun/rubbershot=4,/obj/item/ammo_casing/shotgun/stunslug=35,/obj/item/ammo_casing/shotgun/ion=14,/obj/item/ammo_casing/shotgun/laserslug=35,/obj/item/ammo_box/speedloader/shotgun=8,/obj/item/ammo_box/magazine/lr30mag=12,/obj/item/ammo_box/magazine/enforcer=18,/obj/item/ammo_box/magazine/enforcer/lethal=28,/obj/item/ammo_box/magazine/sp8=8,/obj/item/ammo_box/magazine/laser=12,/obj/item/ammo_box/magazine/wt550m9=20,/obj/item/ammo_box/magazine/m556=12,/obj/item/ammo_box/a40mm=4,/obj/item/ammo_box/c46x30mm=8,/obj/item/ammo_box/inc46x30mm=4,/obj/item/ammo_box/tox46x30mm=4,/obj/item/ammo_box/ap46x30mm=4,/obj/item/ammo_box/laserammobox=4);
```
`/item/grenade/flashbangs` нету в билде. Есть коробка с этими гренами.

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Remove BSA fun

Снес стекло, которое потенциально могло бы помочь боргам стрелять из БСА, находясь в ресторанобарокафе

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Merge branch 'СС-arm2' of https://github.com/ItsMeJaba/Paradise into СС-arm2

ItsMeJaba committed:
Мержконфликты

Слава ганпорно, больше подтипов 1 пушки ради скина, БОЛЬШЕ

PinnoQYow commented:
Шаттерсы под чьим доступом? На нынешнем ЦК можно спокойненько на Эмбере залутать админскую карточку в шкафу, вырезав шлюз, а потом с её помощью и гибпушку в офисе ОСО.
![image](https://github.com/ss220-space/Paradise/assets/155892633/3d130493-7da6-4290-bd01-3838aefd91b8)

ItsMeJaba commented:
> Шаттерсы под чьим доступом? На нынешнем ЦК можно спокойненько на Эмбере залутать админскую карточку в шкафу, вырезав шлюз, а потом с её помощью и гибпушку в офисе ОСО. ![image](https://private-user-images.githubusercontent.com/155892633/304294201-3d130493-7da6-4290-bd01-3838aefd91b8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc4MzMzNDUsIm5iZiI6MTcwNzgzMzA0NSwicGF0aCI6Ii8xNTU4OTI2MzMvMzA0Mjk0MjAxLTNkMTMwNDkzLTdkYTYtNDI5MC1iZDAxLTM4MzhhZWZkOTFiOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxM1QxNDA0MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03OTQ3MjI3ZTIxNDQyN2FiZjMzZTVlNjI2YWRjZGI2MzA2YzZjOGNhNDI5NzQ3MjYwNmI3MmJiMTI4MWJiNDQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.OJnyoMzw4JmVpjhrin_by8-98NusRLE7MOIJQ2KWuLw)

req_access = list(114) - доступ ЦКшных ролей, такой же как и на всех остальных дверях в офисы, т.е. не ЕРТшный (109/111). Это шаттерсы специально от говнистостых

SAADf603 commented:
ревью через час

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Little nit of hehe

Правки после ж0ского ревью бригадира

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

TrashDoxx commented:
Накладываю вето.

ItsMeJaba commented:
> Накладываю вето.

Пральна, у меня последний коммит не залит с дофиксами

TrashDoxx commented:
> Пральна, у меня последний коммит не залит с дофиксами

Пингани, как зальёшь.

ItsMeJaba committed:
LastFIxes

Заменил открытые говнокодом VV шаттерсы на аналог с говнокода
Нашел потерянные пару тайлов под движками ферри (всех 16)

TrashDoxx commented:
Снимаю вето. @Dimach Мержи, это баг-фикс.

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Ваши икон заборонено

Апдейт иконок не дал мне своровать спрайты для моих костыльно-кастомных вендоматов

SAADf603 reviewed (changes requested):
Ссылка на ревью: https://discord.com/channels/617003227182792704/943940908162875473/1214227998388002816

ItsMeJaba committed:
Sho to

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Актуализация

Я просто вырезал кусок с НТ ЦК, внес изменения за эти... 5 месяцев? И воткнул на актуальную карту

ItsMeJaba committed:
Актуализация

Я просто вырезал кусок с НТ ЦК, внес изменения за эти... 5 месяцев? И воткнул на актуальную карту

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Merge branch 'СС-arm2' of https://github.com/ItsMeJaba/Paradise into СС-arm2

ItsMeJaba committed:
Update CentComm.dmm

Vladisvell reviewed (dismissed):
Ты не ты когда ты
mapper_angry.ogg
mapper_annoyed.ogg
mapper_death.ogg
mapper_disappointed.ogg
mapper_scream.ogg

NightDawnFox reviewed (dismissed):
Ждет мерджа с момента Nov 18, 2023

ItsMeJaba committed:
Fix: overlays & smooth criminal

Поправил оверлеи добавленных вендоматов, поправил шаттерсы-стены-костыли над туррелями

ItsMeJaba committed:
logo fix

Из-за наслоения прерывался спрайт надписи

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
fix: bump tele

После рефактора звея с выпилом прока бампа он начал есть фальш-стены, Д2 больше не покинет оружейной Кеннеди

ItsMeJaba committed:
"Фикция"

Доебы мисстера SQUEEK учтены

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Refucktor

Спасибо за рефактор конвейеров, было невкусно и грустно

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Мои любимые рефаторы с изменениями пути

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Мерге конфликт

Как же я люблю апдейты на +-/слово/ в пути

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2

ItsMeJaba committed:
Merge branch 'master220' into СС-arm2
